components/channel_header_dropdown/channel_header_dropdown_items.js

```diff
@@ -267,7 +283,9 @@ export default class ChannelHeaderDropdown extends React.PureComponent {
                         />
                     </ChannelPermissionGate>
                 </Menu.Group>
-
+                <Menu.Group>
+                    {pluginItems}
+                </Menu.Group>
                 <Menu.Group divider={divider}>
                     {isMobile &&
                         <MobileChannelHeaderPlug
```

Comment:
Unrelated, but I can't remember what this is -- does this overlap with any of your effort?

Reply:
I had to look into that myself while I was in there. It seems that the `MobileChannelHeaderPlug` component is used to display the plugins channel header buttons (not to be confused with the channel header menu) in mobile view. 
However, I honestly do not understand why it's being used here too. Maybe @jespino has some background on this.